includes/api/class-wc-api-products.php

```diff
@@ -1420,59 +1420,36 @@ function attributes_cmp( $a, $b ) {
 
 		} else {
 
-			// Regular Price.
```

Comment:
The punctuation was inconsistent between v2 and v3 api so I wasn't shure what the current way to go is. Sorry about that.